cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -0,0 +1,39 @@
+<form>
+    <h3 class="modal-section-title"><%= gettext('Timed Exam') %></h3>
+    <div class="modal-section-content has-actions">
+        <div class='exam-time-list-fields'>
+            <ul class="list-fields list-input">
+                <li class="field field-checkbox checkbox-cosmetic">
+                    <input type="checkbox" id="id_timed_examination" name="timed_examination" class="input input-checkbox" />
+                    <label for="id_timed_examination" class="label">
+                      <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
+                      <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
+                      <%= gettext('This exam is timed') %>
+                    </label>
+                </li>
+            </ul>
+        </div>
+        <div class='exam-time-list-fields'>
+            <ul class="list-fields list-input time-limit">
+                <li class="field field-text field-time-limit">
+                    <label for="id_time_limit" class="label"><%= gettext('Time Allotted (HH:MM):') %></label>
+                    <input type="text" id="id_time_limit" name="time_limit"
+                      value=""
+                      placeholder="HH:MM" class="time_limit release-time time input input-text" autocomplete="off" />
+                </li>
+                <p class='field-message'><%= gettext('Students will be warned before time expires') %></p>
```

Comment:
"Students see a warning {X} minutes before time expires. To allow a specific student additional time to complete this exam, you can create an allowance in the Proctoring tab in the LMS."